server.js

```diff
@@ -578,6 +578,41 @@ camp.route(/^\/sonar\/(http|https)\/(.*)\/(.*)\/coverage\.(svg|png|gif|jpg|json)
       });
     }));
 
+// Coverity integration
+camp.route(/^\/coverity\/(.+)\.(svg|png|gif|jpg|json)$/,
```

Comment:
In case coverity offers more than one product in the future, can you use `/coverity/scan/` or something like that?